hokusai/production.yml

```diff
@@ -184,9 +184,16 @@ spec:
             - name: fix-the-volume-permission
               image: busybox
               command:
-              - sh
+              - /bin/sh
               - -c
-              - chown -R 1001:1001 /home/deploy/data
+              - |
```

Comment:
Really cool how you can write multiline bash command like this inline. I think it's fine as is since its not very long.

Reply:
Nice. Yeah I agree this seems like the best place. I hope our future isn't filled with many bash scripts embedded in YAML files though!